test/icc-x-api/icc-maintenance-task-x-api-test.ts

```diff
@@ -36,6 +37,10 @@ const hcp2UserName = process.env.HCP_2_USERNAME!
 const hcp2Password = process.env.HCP_2_PASSWORD!
 const hcp2PrivKey = process.env.HCP_2_PRIV_KEY!
 
+const hcp3UserName = process.env.HCP_3_USERNAME!
```

Comment:
Add those 3 variables into the .github/workflows/build.yml file for SonarCloud job